.github/workflows/update-helm-repo.yaml

```diff
@@ -195,10 +196,10 @@ jobs:
 
             Tag on source: https://github.com/${{ github.repository }}/releases/tag/${{ steps.parse-chart.outputs.tagname }}
           files: |
-            ${{ env.CR_PACKAGE_PATH }}/${{ steps.parse-chart.outputs.tagname }}.tgz
-            ${{ env.CR_PACKAGE_PATH }}/${{ steps.parse-chart.outputs.tagname }}.tgz.prov
+            ${{ env.CR_PACKAGE_PATH }}/${{ steps.parse-chart.outputs.packagename }}.tgz
+            ${{ env.CR_PACKAGE_PATH }}/${{ steps.parse-chart.outputs.packagename }}.tgz.prov
           repository: grafana/helm-charts
-          tag_name: ${{ steps.parse-chart.outputs.tagname }}
+          tag_name: ${{ steps.parse-chart.outputs.packagename }}
```

Comment:
shouldn't that remain as `tagname`?

Reply:
that's exactly my question to @trevorwhitney  :) I don't know what the intention was (for mimir the two are the same)

Reply:
i asked because the git tag uses tagname
```
git tag "${{ steps.parse-chart.outputs.tagname }}"
```
if we use a different value on the release than the tag name, will that work?

Reply:
so the intention is to not create a tag `loki-v3.0.0`, as it confused people, making them think that was a new version of `loki` being released. We want the tag to be prefixed with `helm` to not confuse people. Mimir doesn't have this problem because your chart is called `mimir-distributed`.

I think we need to keep this as `tagname` as @dimitarvdimitrov pointed out.